src/components/atoms/button/button.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
I had intended on using this component as part of the search-input molecule. Need to figure more details out but without a more formalized design system this wasn't a good move at the time.